From b111475f8c0ad8bb90fc49a6169d169148a13bea Mon Sep 17 00:00:00 2001
From: Dan Walkes <danwalkes@trellis-logic.com>
Date: Sun, 19 Feb 2023 01:05:54 -0700
Subject: [PATCH] Add version remap script support

Since NGC doesn't include tags for every L4T revision,
add support for a remap script which can remap L4T_VERSION
to one which is supported by NGC.

Signed-off-by: Dan Walkes <danwalkes@trellis-logic.com>
---
 scripts/docker_test_ml.sh | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/scripts/docker_test_ml.sh b/scripts/docker_test_ml.sh
index a71f279..73dee92 100755
--- a/scripts/docker_test_ml.sh
+++ b/scripts/docker_test_ml.sh
@@ -11,6 +11,10 @@ if [ -e scripts/l4t_version_remap.sh ]; then
         source scripts/l4t_version_remap.sh
 fi
 
+if [ -e scripts/l4t_version_remap.sh ]; then
+        source scripts/l4t_version_remap.sh
+fi
+
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 TEST_MOUNT="$ROOT/../test:/test"
 CONTAINERS=${1:-"all"}
-- 
2.34.1

